README.md

```diff
@@ -72,22 +96,25 @@ As a result, the first build can take a very long time depending on the server l
 
 ## Run RMF Demos
 
-Demonstrations of RMF is shown in [rmf_demos](https://github.com/open-rmf/rmf_demos/)
+Demonstrations of RMF is shown in [rmf_demos](https://github.com/open-rmf/rmf_demos/).
 
 ### Docker Containers
 Alternatively, you can run RMF Demos by using [docker](https://docs.docker.com/engine/install/ubuntu/).
 
 Pull docker image from `open-rmf/rmf` github registry (setup refer [here](https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-docker-for-use-with-github-packages#authenticating-with-a-personal-access-token)).
+
 ```bash
 docker pull docker.pkg.github.com/open-rmf/rmf/rmf_demos:latest
 docker tag docker.pkg.github.com/open-rmf/rmf/rmf_demos:latest rmf:latest
```

Comment:
According to https://github.com/open-rmf/rmf/issues/104, can update these 2 lines too
```
docker pull ghcr.io/open-rmf/rmf/rmf_demos:latest
docker tag ghcr.io/open-rmf/rmf/rmf_demos:latest rmf:latest
```